managed/utils/encryption/encryption.go

```diff
@@ -74,7 +75,16 @@ type QueryValues struct {
 // New creates an encryption; if key on path doesn't exist, it will be generated.
 func New() *Encryption {
 	e := &Encryption{}
-	e.Path = encryptionKeyPath()
+	customKeyPath := os.Getenv("PMM_ENCRYPTION_KEY_PATH")
+	if customKeyPath != "" {
+		e.Path = customKeyPath
+	} else {
+		if runtime.GOOS == "darwin" { // for development on macOS
```

Comment:
Should this be done via a `PMM_DEV_` variable?

Reply:
I think it will complicate logic and we already have `PMM_ENCRYPTION_KEY_PATH` above for that, maybe there should be another option to check if it's used in test